telegram/ext/jobqueue.py

```diff
@@ -387,7 +445,8 @@ def __init__(self,
                  days=Days.EVERY_DAY,
                  name=None,
                  job_queue=None,
-                 tzinfo=_UTC):
+                 tzinfo=_UTC,
+                 is_monthly=None):
```

Comment:
Please add `is_monthly` to the docstring (both as argument and attribute)

Reply:
done